src/chi-vue/src/components/data-table-save-view/SaveView.tsx

```diff
@@ -87,12 +86,26 @@ export default class SaveView extends Vue {
     this.$emit(SAVE_VIEW_EVENTS.HIDE);
   }
 
+  disableSaveButton() {
+    if (this.saveButtonDisabled !== undefined) {
+      return this.saveButtonDisabled;
```

Comment:
if `saveButtonDisabled` is equal to `false` **Save button** always is active(clickable)
(even if `this.viewTitle.length === 0` or `this.viewTitle === this.$props.config.title && this.isDefaultChecked === !!this.$props.default`)

just wonder if it's expected behaviour

Reply:
The idea here is to give the developer **full control** of the `disabled` attribute of the save button, so if they set this property (true/false) they can skip our logic and make their own and if they don't use it, the `disabled` will work with our logic by default.